hawkbit-rest/hawkbit-ddi-resource/src/test/java/org/eclipse/hawkbit/ddi/rest/resource/AbstractDDiApiIntegrationTest.java

```diff
@@ -234,4 +244,86 @@ private static String getDownloadAndUploadType(final Action.ActionType actionTyp
         return "attempt";
     }
 
+    protected String getJsonRejectedCancelActionFeedback() throws JsonProcessingException {
+        return getJsonActionFeedback(DdiStatus.ExecutionStatus.REJECTED, DdiResult.FinalResult.SUCCESS,
+                Collections.singletonList(RandomStringUtils.randomAlphanumeric(1000)));
```

Comment:
Does it make sense to provide some more meaningful messages instead of random ones, e.g.: "proceeding", "closed", "rejected", etc.?

Reply:
Changed its value to meaningful messages as suggested by you